src/app/utils/bibDetailsUtils.jsx

```diff
@@ -212,6 +212,73 @@ const constructSubjectHeadingsArray = (url = '') => {
     });
 };
 
+/**
+ * isRtl(string)
+ * Returns true (or false) depending on whether the string needs to be read right to left
+ * @param {string} string
+ * @return  {boolean}
+ */
+const isRtl  = (string) => {
+  if (typeof string !== 'string') { return null }
+  return string.substring(0, 1) === '\u200F'
```

Comment:
Can you include what this character is and why it's being used to verify if a string is rtl?

Reply:
I've expanded the comment to address this